lib/reporters/base.js

```diff
@@ -219,6 +219,11 @@ exports.list = function(failures) {
     stack = stack.replace(/^/gm, '  ');
 
     console.log(fmt, (i + 1), test.fullTitle(), msg, stack);
+
+    // [hack] Handle waterline's nested `originalError`
+    if (err.originalError && err.originalError instanceof Error) {
```

Comment:
Curious, is `originalError` ever not an instance of `Error`? Would we want to console log it anyways if it weren't?